dev/yarn.Dockerfile

```diff
@@ -2,4 +2,3 @@ FROM node:latest
 
 WORKDIR /app/server
 RUN yarn install
```

Comment:
We need to copy the appropriate package.json file here, right?

Reply:
I'm not sure... it works without that line, I assume because of the volume mounting

Reply:
`yarn install` won't install anything, since the docker build step doesn't do the mounting; the docker run step does do the mounting, but then the command we send is `yarn server`, but we don't have any yarn packages yet.  So if you run `yarn build` outside of the process once it works, but not if you don't.  The command should probably be `yarn install && yarn serve` and then remove the `yarn install` from the docker file, since it won't ever do anything useful.

Reply:
This shows up with a fresh install but not if you've already done a yarn install in that directory.

Reply:
In that case, I think we can get rid of `yarn.Dockerfile` since all of its configuration can be done in `docker-compose.override.yml`

Reply:
Changed in [b43e302](https://github.com/girder/shapeworks-cloud/pull/332/commits/b43e3020dcbdd1822fef8732ef824bbba173da8c)